sklearn/ensemble/forest.py

```diff
@@ -89,7 +91,11 @@ def _parallel_build_trees(tree, forest, X, y, sample_weight, tree_idx, n_trees,
         curr_sample_weight *= sample_counts
 
         if class_weight == 'subsample':
-            curr_sample_weight *= compute_sample_weight('auto', y, indices)
+            with warnings.catch_warnings():
+                warnings.simplefilter('ignore', DeprecationWarning)
+                curr_sample_weight *= compute_sample_weight('auto', y, indices)
```

Comment:
It is raised in ``_validate_y_class_weight`` here: https://github.com/scikit-learn/scikit-learn/pull/4347/files#diff-65dc1c804f310acfd90f5ea83286065cR423